### PR TITLE
minor code refactor

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

File: pangeo_forge_orchestrator/model_builders.py [hunk 1]
```diff
@@ -122,10 +122,10 @@ def make_table_cls(self) -> SQLModel:
         cls_name = self.make_cls_name(self.base, "")
         attrs = dict(id=Field(default=None, primary_key=True))
         annotations = dict(id=Union[int, None])
-        attrs.update(dict(__annotations__=annotations))
+        attrs |= dict(__annotations__=annotations)
```
Comment (rabernat) [inline pangeo_forge_orchestrator/model_builders.py:125]: What does this operator do? Is it the same as update? It's new to me but happy to learn something new! 😄

File: pangeo_forge_orchestrator/routers/github_app.py [hunk 2]
```diff
@@ -615,7 +612,7 @@ async def run(
         recipe_run.status = "in_progress"
         # Start time was first set when recipe run was queued, which could have been ages ago,
         # so if we don't update it now, we won't capture how long the pipeline actually took.
-        recipe_run.started_at = datetime.utcnow().replace(microsecond=0)
+        recipe_run.started_at = datetime.now(timezone.utc).replace(microsecond=0)
```
Comment (rabernat) [inline pangeo_forge_orchestrator/routers/github_app.py]: Why is this better than utcnow?

**Review by rabernat — COMMENTED** [review]
Thanks for doing this Anderson. I have a few minor questions for my understanding.

File: pangeo_forge_orchestrator/model_builders.py (hunk 1 above)
Comment (andersy005) [reply pangeo_forge_orchestrator/model_builders.py:125]: yes, this is a shorthand for dict update: https://peps.python.org/pep-0584/... i believe it's available for py39+.... happy to revert to the more explicit `dict.update`

File: pangeo_forge_orchestrator/routers/github_app.py (hunk 2 above)
Comment (andersy005) [reply pangeo_forge_orchestrator/routers/github_app.py]: This just adds the timezone info in the string repr. However, It turns out this new version introduces a breaking change. So, i reverted back to the original version 

```python
In [4]: datetime.utcnow().replace(microsecond=0).isoformat()
Out[4]: '2022-10-24T23:52:36'
```

```python
In [7]: datetime.now(timezone.utc).replace(microsecond=0).isoformat()
Out[7]: '2022-10-24T23:53:27+00:00'
```

**Comment by rabernat** [comment]
I'm not sure I'd call this a refactor. Mostly just cosmetic changes to the code. Thanks a lot for this PR Anderson.

**Comment by andersy005** [comment]
Thank you @rabernat! I concur. This PR didn't have much in it. I probably should have opened this PR in a draft mode ( I was planning to add more commits to it 🙂). I'll open a separate PR later

**Comment by rabernat** [comment]
Sorry for the premature merge. 🤦  Yes if something is not a draft, I generally assume it is considered ready to be merged.

**Comment by cisaacstern** [comment]
Lots of great stylistic improvements here, and I definitely learned some things, including `|=`.

Thanks, @andersy005!

**Pull request merged**